invenio_cli/helpers/cookiecutter_wrapper.py

```diff
@@ -41,11 +41,11 @@ def repository(self):
         if self.flavour.upper() == 'RDM':
             self.template_name = 'cookiecutter-invenio-rdm'
             repo = {
-                'template': 'https://github.com/inveniosoftware/' +
+                'template': 'https://github.com/zzacharo/' +
                             '{}.git'.format(self.template_name),
                 # set to cookiecutter release version
                 # reset to master in development
-                'checkout': 'master'
+                'checkout': 'react-templates'
```

Comment:
I think you didn't mean to commit that file :)

Reply:
oups :P